src/tests/compiler/clvm.rs

```diff
@@ -213,3 +214,19 @@ fn test_random_int_just_the_conversion_functions_and_no_other_things_from_the_st
         }
     }
 }
+
+#[test]
+fn test_sha256_tree_hash() {
+    let filename = "resources/tests/assert.clvm.out";
+    let assert_clvm_compiled = fs::read_to_string(filename).expect("should exist");
+    let parsed = parse_sexp(
+        Srcloc::start(filename),
+        assert_clvm_compiled.as_bytes().iter().copied(),
+    )
+    .expect("should parse");
+    let hash_result = Bytes::new(Some(BytesFromType::Raw(sha256tree(parsed[0].clone())))).hex();
```

Comment:
it seems to me this could be simplified

Reply:
I'll give some thought to simplifying the test cases.